docs/content/ingestion/index.md

```diff
@@ -100,8 +100,9 @@ This is for realtime ingestion. Make sure to include "io.druid.extensions:druid-
 | type | String | This should say `avro_stream`. | no |
 | avroBytesDecoder | JSON Object | Specifies how to decode bytes to Avro record. | yes |
 | parseSpec | JSON Object | Specifies the format of the data. | yes |
+| schemaMappings | JSON Object | Specifies a mapping from field names found within the parseSpec to data within a complex avro structure. | no |
```

Comment:
Yes, the `metricsSpec` differentiates between `name` and `fieldName`, and note that the `schemaMapping` works for `metricsSpec` too, so if `schemaMapping` like this:
```json
"schemaMappings":{
    "number" : "field.number"
},
```
we need to use `metricsSpec` like this:
```json
"metricsSpec" : [
  {"type" : "doubleSum",
    "name" : "number",
    "fieldName" : "number"
  }
]
```
so, towards dimensions we get one-level-mapping and pulling-value-from-complex-avro-data while metrics we get two-level-mapping and pulling-value-from-complex-avro-data, would it letting them consistent make sense? 

Maybe the `schemaMapping` can be added into `DimensionsSpec` as an alternative for `dimensions`?